Sensors/SMCDellSensors/SMIMonitor.hpp

```diff
@@ -202,6 +202,11 @@ class SMIMonitor : public OSObject
 	 */
 	IOSimpleLock *queueLock {nullptr};
 	
+	/**
+	 *  A simple lock to disable preemtion
```

Comment:
I think it should be "preemption" instead of "preemtion".

Reply:
Thank you, it has been fixed.